### PR TITLE
[Term Entry] C++ vector .back() 

### PR DESCRIPTION
**Pull request by SaviDahegaonkar**
### Description
New term entry on C++ vector. back( )

### Issue Solved
#4984 

### Type of Change

- Adding a new entry

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by SaviDahegaonkar** [comment]
Hey @avdhoottt 
Can you close this PR as I have re-raised this PR ?

**Comment by avdhoottt** [comment]
Got it.

File: content/cpp/concepts/vectors/terms/back/back.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```
+The above code outputs:
+```shell
+The last element in the vector is: 60
+The last element in the vector after modification is: 80
+```
+
+## Codebyte Example
+
+This codebyte example shows that how a last element in the vector in accessed and modified using the `back()` method.
```
Comment (avdhoottt) [inline content/cpp/concepts/vectors/terms/back/back.md:50]:
```suggestion
This codebyte example shows how a last element in the vector is accessed and modified using the `back()` method:
```

**Review by avdhoottt — APPROVED** [review]
LGTM!!

**Pull request merged**